modules/aoc/aocCommand.ts

```diff
@@ -12,7 +13,8 @@ export default {
         description: "Returns the Advent of Code Leaderboard"
     },
     handler: async function(interaction: CommandInteraction) {
-        const embed = await embedLeaderboard(get("id", "aoc") as number, get("year", "aoc") as number);
+        const now = DateTime.now().setZone("Europe/Berlin");
+        const embed = await embedLeaderboard(get("id", "aoc") as number, now.year);
```

Comment:
it would be nice to have an optional parameter for the year too. If not set it would take the current year. If set it takes the given year